scripts/utils.sh

```diff
@@ -76,4 +76,11 @@ function wait_for_url_and_run() {
     fi
 }
 
+function close_external_ports() {
+    sudo firewall-cmd --zone=public --permanent --remove-port=6000/tcp
```

Comment:
The removal of the ports should match the addition. Remove the `--permanent` flag